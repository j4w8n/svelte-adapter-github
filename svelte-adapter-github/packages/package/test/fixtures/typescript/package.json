{
	"name": "typescript",
	"private": true,
	"version": "1.0.0",
	"description": "standard typescript package",
	"type": "module"
}
