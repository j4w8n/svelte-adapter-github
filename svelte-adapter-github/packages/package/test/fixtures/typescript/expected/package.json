{
	"name": "typescript",
	"private": true,
	"version": "1.0.0",
	"description": "standard typescript package",
	"type": "module",
	"exports": {
		"./package.json": "./package.json",
		"./Plain.svelte": "./Plain.svelte",
		"./Test.svelte": "./Test.svelte",
		"./Test2.svelte": "./Test2.svelte",
		"./utils": "./utils.js",
		".": "./index.js"
	},
	"svelte": "./index.js"
}